### ISSUE TITLE
"An unexpected situation arose"

### ISSUE THREAD

**Issue by alex-r-bigelow** [rollup/rollup-plugin-babel#243]
I'm trying to adapt my build to the new Babel 7 release, and ran across this cryptic error:

```
[!] (babel plugin) Error: An unexpected situation arose. Please raise an issue at https://github.com/rollup/rollup-plugin-babel/issues. Thanks!
src/main.js
Error: An unexpected situation arose. Please raise an issue at https://github.com/rollup/rollup-plugin-babel/issues. Thanks!
    at error (/media/data/Repositories/mure-library/node_modules/rollup/dist/rollup.js:3438:30)
    at Object.error (/media/data/Repositories/mure-library/node_modules/rollup/dist/rollup.js:21120:17)
    at /media/data/Repositories/mure-library/node_modules/rollup-plugin-babel/src/preflightCheck.js:61:9
    at Object.transform$1 (/media/data/Repositories/mure-library/node_modules/rollup-plugin-babel/src/index.js:77:20)
    at /media/data/Repositories/mure-library/node_modules/rollup/dist/rollup.js:20831:25
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:266:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:596:3)
```
Potentially relevant configuration files:
- [rollup.config.js](https://github.com/mure-apps/mure-library/blob/b83caa7371577bbc607ae508cb9f16a5b628c2c9/rollup.config.js)
- [package.json](https://github.com/mure-apps/mure-library/blob/b83caa7371577bbc607ae508cb9f16a5b628c2c9/package.json)
- [.babelrc](https://github.com/mure-apps/mure-library/blob/b83caa7371577bbc607ae508cb9f16a5b628c2c9/.babelrc)

I still consider myself a babel / rollup n00b, so there's a very good chance I'm doing something very wrong :wink:

**Comment by Comandeer** [comment]
I encountered the same bug, after updating https://github.com/Comandeer/babel-preset-rollup to Babel 7 and `@babel/plugin-external-helpers`. Seems like the error is connected with this update.

**Comment by Andarist** [comment]
Could u try `rollup-plugin-babel@next`? Also usage with `@babel/plugin-external-helpers` is deprecated now (if u dont want REAL `@babel/plugin-external-helpers` behaviour, if u really want it - but it will rely on non bundled global variable - u have to pass `externalHelpers: true` to the plugin options) - we do the same thing (helpers deduplicating) that was previously achieved in combination with external-helpers plugin automatically

**Comment by alex-r-bigelow** [comment]
Removing the external helpers seems to have gotten rid of the error. Not sure whether this issue is still relevant for those that need them.

**Comment by flyyang** [comment]
Same issue here

**Comment by Andarist** [comment]
@flyyang could u check out `rollup-plugin-babel@next`?

**Comment by Andarist** [comment]
I've released a fix for this in 4.0.3
